src/Config/Config.ts

```diff
@@ -323,7 +323,7 @@ export class BridgeWidgetConfig {
         if (yaml.disallowedIpRanges !== undefined && (!Array.isArray(yaml.disallowedIpRanges) || !yaml.disallowedIpRanges.every(s => typeof s === "string"))) {
             throw new ConfigError("widgets.disallowedIpRanges", "must be a string array");
         }
-        if (typeof yaml.publicUrl !== "string") {
+        if (typeof yaml.publicUrl !== "string" || !yaml.publicUrl) {
```

Comment:
Minded to say we should go a step further and try to parse the URL with `new URL()`